cnxepub/tests/test_collation.py

```diff
@@ -280,4 +280,4 @@ def test_with_ruleset(self):
         # Check for the appended composite document
         self.assertEqual(len(collated_binder), 3)
         self.assertEqual(collated_binder[2].metadata['title'],
-                         'Composite One')
+                         'Document One')
```

Comment:
Hm... I'm not sure if I understand this commit.  Why are we not getting the titles from the TOC as the title overrides?